provider/label/partial.go

```diff
@@ -60,6 +60,39 @@ func GetRedirect(labels map[string]string) *types.Redirect {
 	return nil
 }
 
+// GetTLSClientCert create tls client header configuration from labels
```

Comment:
`// GetTLSClientCert creates TLS client header configuration from labels` instead of `// GetTLSClientCert create tls client header configuration from labels`